### PR TITLE
[FSDE-14187] Data table example updates for complex/actions

### PR DESCRIPTION
**Pull request by soujanreddy**
1. Position Kebab icon dropdown in Actions column left-start in Data tables. (Complex example and localhost:9090).
2. Replace info icon popover content to : “Helpful information goes here.“.
3. Improve disposing of action column dropdowns (Implemetation + code snippets).
4. Adjust localhost:9090 client side column labels nomenclature:
String → Product
Date → Created
Product/Service ID → Service ID

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b54f7425d33a9b08fe88c4a434fa8d7068cb0e96-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/website/views/components/data-table/examples/portal/_complex.pug [hunk 1]
```diff
@@ -10,15 +10,8 @@ h3 Complex
...
             :data="table",
           )
-            template(#statusdesc='payload')
-              i(:class="`chi-icon icon-${payload.due.icon} -icon--${payload.due.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.due.status }}
-              br
-              i(:class="`chi-icon icon-${payload.overdue.icon} -icon--${payload.overdue.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.overdue.status }}
             template(#status="payload")
-              i(:class="`chi-icon icon-${payload.icon} -icon--${payload.color}`" aria-hidden="true")
```
Comment (janibolkvadze) [inline src/website/views/components/data-table/examples/portal/_complex.pug:20]: Why do we remove the template here? This makes the column cells render empty. The idea was to simplify the column description only.
Also, can you check what's wrong with the code snippet wrapper? It renders like this for me.
Screenshot 2022-02-25 at 16 19 09

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/2/. :x:

File: src/website/views/components/data-table/examples/portal/_complex.pug (hunk 1 above)
Comment (soujanreddy) [reply src/website/views/components/data-table/examples/portal/_complex.pug:20]: My bad! I have updated it as required. This should be resolved now.

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://670e3d94b91ff81b162640c7859cb5424d88aa36-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/website/views/components/data-table/examples/portal/_complex.pug [hunk 2]
```diff
@@ -11,11 +11,7 @@ h3 Complex
             :data="table",
           )
             template(#statusdesc='payload')
-              i(:class="`chi-icon icon-${payload.due.icon} -icon--${payload.due.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.due.status }}
-              br
-              i(:class="`chi-icon icon-${payload.overdue.icon} -icon--${payload.overdue.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.overdue.status }}
+              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.statusInfo }}
```
Comment (janibolkvadze) [inline src/website/views/components/data-table/examples/portal/_complex.pug:14]: To render column description popover as a simple string, there is no need of having a custom template for it.
```
{
  head: {
    [code: string]: {
      label: string;
      sortable?: boolean;
      sortBy?: string;
      sortDataType?: 'string' | 'number' | 'date' | 'boolean';
      align?: 'left' | 'center' | 'right';
      allowOverflow?: 'visible' | 'hidden';
      key?: boolean;
      bold?: boolean;
      description?: string | DataTableColumnDescription; // description can be just a string. No need for custom payload + template.
      isPrintDisabled?: boolean;
    };
  };
  body: DataTableRow[];
}
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]
https://670e3d94b91ff81b162640c7859cb5424d88aa36-chi-pr.enterprise.ctl.io/components/data-table/?theme=Portal#props

File: src/website/views/components/data-table/examples/portal/_complex.pug (hunk 2 above)
Comment (soujanreddy) [reply src/website/views/components/data-table/examples/portal/_complex.pug:14]: @janibolkvadze This particular example is not utilizing the types which you have mentioned. I have tried adding description but it renders on the table header.
SS

Comment (janibolkvadze) [reply src/website/views/components/data-table/examples/portal/_complex.pug:14]: Screenshot 2022-03-08 at 11 17 44

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b151492fa331df6c9a0211afca544f2234593304-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b151492fa331df6c9a0211afca544f2234593304-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/website/views/components/data-table/examples/portal/_complex.pug (hunk 2 above)
Comment (soujanreddy) [reply src/website/views/components/data-table/examples/portal/_complex.pug]: Thank you for clarifying.

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bd1d8bad616e31bbed3511a5094d9dc7129c4713-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/7/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://3aab23121102d29fd952c4eab3d4e5bdb5c2927b-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

File: src/website/views/components/data-table/examples/portal/_complex.pug [hunk 3]
```diff
@@ -76,10 +70,7 @@ h3 Complex
             <!-- Vue component -->
             <ChiDataTable :config="config" :data="table">
               <template #statusdesc="payload">
-                <div v-for="status in payload" :key="status.status">
-                  <i :class="`chi-icon icon-${status.icon} -icon--${status.color}`" aria-hidden="true"></i>
-                  <span class="-text--truncate" style="padding-left: 0.5rem;">{{ status.status }}</span>
-                </div>
+                <span class="-text--truncate" style="padding-left: 0.5rem;">{{ payload.statusInfo }}</span>
               </template>
```
Comment (jllr) [inline src/website/views/components/data-table/examples/portal/_complex.pug:74]: @soujanreddy this template is not necessary. Please remove it.

**Review by jllr — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-335/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://0910c8a7cf333ae8fec3496f441582e9db94f904-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-335-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**